README.md

```diff
@@ -95,9 +95,22 @@ A single directory is mounted at `/job` in each container before it is run, and
 
 This helps facilitate cross-stage communication, which becomes particularly useful if certain stages need to pass along results.
 
+### Private Registry Support
+
+If you choose to [set up a private registry](https://docs.docker.com/registry/deploying/), you should secure it. After it is secured, set up your credentials in the docker daemon using 
+
+```shell
+docker login [registry.example.com]
```

Comment:
`docker login` does not "set up" the credentials right? I think the entire process of setting credentials and adding auth to the registry is documented here: https://docs.docker.com/registry/deploying/#native-basic-auth

Reply:
What you linked to is part of the deployment process for a registry, which I don't think needs to be part of our README. `docker login` sets up the credentials for the daemon.